### DISCUSSION TITLE
Demo using Streamlit

### DISCUSSION THREAD

**Discussion by straussmaximilian** [sinaptik-ai/pandas-ai#59] [Show and tell]
Hi,
This library is great! I made a small streamlit interface for pandas-ai:
https://github.com/straussmaximilian/pandas-ai-streamlit

Streamlit has free hosting for open-source packages, so an instance is running [here](https://straussmaximilian-pandas-ai-streamlit-app-sy55b1.streamlit.app/)

Would this be useful to integrate into the repo? Streamlit would add one additional dependency, though, so this is why I made it stand-alone for now. Anyhow, happy to make a PR if this would be useful.

Best,

Max
![streamlit-app-2023-05-06-01-05-26](https://user-images.githubusercontent.com/16521288/236584133-0bb64996-ae04-47f6-84db-55c9e8039f5c.gif)

**Comment by gventuri** [comment]
Hey @straussmaximilian, impressive project, well done, I also tested it out and really liked it!
However, we'd love to keep the repo as simple and lightweight as possible, in order to be flexible enough to support all the different use cases.

Great job tho, once again! 👏
